modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -28,6 +30,18 @@ To enable the VPC to host your workers, it must be bound to a business group and
 
 Read the following FAQs to understand how to plan a successful VPC setup.
 
+== FAQ: How to Calculate VPC and VPN Licensing Requirements
+
+To connect a CloudHub VPC through IPSec VPN, you must have the required number of licenses for your deployment scenario. 
+
+The Anypoint Platform base subscription includes two VPC licenses, which entitles your Anypoint Platform organization to two licensed VPCs. VPN licenses are bundled with the VPC license, so each VPC license also entitles the organization to one VPN gateway with a connection to one public IP address in a remote location. 
+
+[NOTE]
+VPN licenses do not need to have a one-to-one relationship with the VPC licenses. For example, you can have two VPCs with two VPNs in one of them and none in the other. 
```

Comment:
"For example, you can have one VPC with two VPNs and no VPNs in the other VPC."?  "in one of them" sounds odd to me.